plugins/window-rules/window-rules.cpp

```diff
@@ -194,11 +195,13 @@ void wayfire_window_rules_t::setup_rules_from_config()
 {
     _rules.clear();
 
-    // Build rule list.
-    auto section = wf::get_core().config.get_section("window-rules");
-    for (auto opt : section->get_registered_options())
+    wf::option_wrapper_t<wf::config::compound_list_t<std::string>> rule_list_option{"window-rules/rules"};
+    auto rule_list = rule_list_option.value();
+
+    for (const auto& [name, rule_str] : rule_list)
     {
-        _lexer.reset(opt->get_value_str());
+        LOGW("Registering ", rule_str);
```

Comment:
Why is this a warning?

Reply:
Oh.. I had added it during testing... What's the function for debug? Would it be LOGD?